src/pages/coc.astro

```diff
@@ -1,5 +1,56 @@
 ---
-import Placeholder from "../components/Placeholder.astro";
+import Layout from "../layouts/BlogPost.astro";
 ---
 
-<Placeholder />
+<Layout
+    title="PyCon Code of Conduct"
+    description="PyCon Code of Conduct"
+    pubDate={new Date("January 01 2024")}
+    heroImage="/blog-placeholder-about.jpg"
```

Comment:
find a relevant hero image, and pub date

Reply:
made banner follow theme color